model/item.cc

```diff
@@ -279,6 +283,13 @@ namespace minsky
     cairo_stroke(cairo);
   }
   
+  bool Item::attachedToDefiningVar() const
+  {
+    for (size_t i=0; i<ports.size(); i++)
```

Comment:
Only port 0 (the output port) is relevant here (except for switches, which have more than one output port, and can be handle via an override).

Reply:
Thanks, this change and the next makes all the items attached to the defining disappear when I toggle var tab display :)

I just have to make them come back when I do the reverse, which they don't for some reason and also deal with int ops which seem to crash Minsky (example exponentialGrowth.mky crashes). After that, this feature should be ready to go and I can work on the second part of it.